src/components/Sum.jsx

```diff
@@ -1 +1,12 @@
-// export function Sum() {}
+import { React } from 'react';
+
+export const Sum = ({ a, b }) => (
+  <p>
+    {`Sum of ${a} and ${b} is ${a + b}`}
+  </p>
+);
+
+Sum.defaultProps = {
+  a: 0,
+  b: 0,
+};
```

Comment:
```suggestion
Sum.defaultProps = {
  a: 0,
  b: 0,
};
```
it works but it would be better if you set default parameters into destructurisation 
```
export const Sum = ({ a = 0, b = 0 })
```